lang/parser/parser.y

```diff
@@ -1494,8 +1510,9 @@ func (yylex *Lexer) cast() *lexParseAST {
 
 // pos is a helper function used to track the position in the lexer.
 func (yylex *Lexer) pos(lval *yySymType) {
-	lval.row = yylex.Line()
-	lval.col = yylex.Column()
+	// add 1 because the lexer starts counting from 0
```

Comment:
IIRC I intentionally left this zero based because it made some code somewhere easier, and only on printf of a message did I add the offsets.

Reply:
Makes sense I guess